src/views/modules/BookItem.js

```diff
@@ -0,0 +1,29 @@
+function BookItem({ id = null, title, category, status, number, price }) {
+  return `
+  <tr>
+    <td>${id}</td>
```

Comment:
why id equals null ? and if id = null then this line will be nothing ?

Reply:
ok let me check it